eng/Subsets.props

```diff
@@ -179,6 +180,10 @@
     <CoreClrProject Include="$(CoreClrProjectRoot)src\tools\crossgen2\crossgen2\crossgen2.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subset.Contains('-diagtools-'))">
```

Comment:
This block needs to be moved above the block at line 159 (the line reference I gave was based on master).